xikolo-ios/ViewControllers/Courses/DropdownViewController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DropdownViewController.swift
+//  xikolo-ios
+//
+//  Created by Bjarne Sievers on 29.10.16.
+//  Copyright © 2016 HPI. All rights reserved.
+//
+
+import UIKit
+
+class DropdownViewController: UIViewController {
```

Comment:
I need the embedding ViewController for the popup on the iPad. The collapse to the small popup is not possible without this wrap and the additional layout constraints.